analysers/analyser_osmosis_railway_angle.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
General note: to enable the analyser for all countries, it must be added to this list too:
https://github.com/osm-fr/osmose-backend/blob/dev/osmose_config.py#L197